test/integration/drivers/virtualbox/create.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+load ${BASE_TEST_DIR}/helpers.bash
```

Comment:
Why not give this test file a more descriptive name?  `create.bats` doesn't tell me much about what it's intended to test.